circle.yml

```diff
@@ -26,6 +26,8 @@ dependencies:
   post:
     - go clean -i net
     - go install -tags netgo std
+    - go get github.com/davecgh/go-spew/spew
+    - go get github.com/pmezard/go-difflib/difflib
```

Comment:
Actually do these need to be go gotten manually?  Wouldn't it be better to update the go get cmd in the makefile to fetch test dependancies, or was that not possible?